src/game/utils.rs

```diff
@@ -1,42 +1,46 @@
-use bevy::prelude::*;
 use crate::game::word_match::components::{Word, WordTarget};
+use bevy::prelude::*;
 
 /**
     Spawn a given word for a given entity, with a black background for contrast
 **/
 pub fn spawn_word(builder: &mut ChildBuilder, word: &str, font: &Handle<Font>) {
-    builder.spawn((
-        Text2dBundle {
-            text: Text::from_sections([
-                TextSection::from_style(TextStyle {
-                    font_size: 20.,
-                    font: font.clone(),
-                    color: Color::RED
-                }),
-                TextSection::new(
-                    word,
-                    TextStyle {
+    builder
+        .spawn((
+            Text2dBundle {
+                text: Text::from_sections([
+                    TextSection::from_style(TextStyle {
                         font_size: 20.,
                         font: font.clone(),
-                        color: Color::WHITE
-                    }
-                )
-            ]),
-            transform: Transform::from_translation(Vec3::new(0., 15., 2.)),
-            ..default()
-        },
-        Word(WordTarget::Enemy(1), word.to_string()))
-    ).with_children(|builder| {
-        builder.spawn(
-            SpriteBundle {
+                        color: Color::RED,
+                    }),
+                    TextSection::new(
+                        word,
+                        TextStyle {
+                            font_size: 20.,
+                            font: font.clone(),
+                            color: Color::WHITE,
+                        },
+                    ),
+                ]),
+                transform: Transform::from_translation(Vec3::new(0., 60., 2.)),
+                ..default()
+            },
+            Word(
+                WordTarget::Enemy(builder.parent_entity().index()),
+                word.to_string(),
+            ),
+        ))
+        .with_children(|builder| {
+            builder.spawn(SpriteBundle {
                 sprite: Sprite {
                     color: Color::BLACK,
+                    // will need to scale this black bar to word size, keep 80 as minimum
```

Comment:
tricky part with this was the healthbar, will likely need to separate it from the black bar to make it look ok